pkgs/os-specific/linux/systemd/bootchart.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, intltool, pkgconfig, libtool
+, libxslt, docbook_xsl, docbook_xml_dtd_42, docbook_xml_dtd_45, systemd }:
+
+assert stdenv.isLinux;
+
+stdenv.mkDerivation rec {
+  version = "230";
+  name = "systemd-bootchart-${version}";
+
+  src = fetchFromGitHub {
+    owner = "systemd";
+    repo = "systemd-bootchart";
+    rev = "v${version}";
+    sha256 = "1dbscx7hsvn6a5rbjfmpskwnp0n2hpwdb8n85gssnlmxas7cw72f";
+  };
+
+  nativeBuildInputs = [
+    autoconf automake intltool pkgconfig libtool libxslt docbook_xsl
+    docbook_xml_dtd_42 docbook_xml_dtd_45
+  ];
+
+  buildInputs = [ systemd ];
+
+  configureFlags = [
+    "--sysconfdir=/etc"
+    "--localstatedir=/var"
+    "--with-rootprefix=$(out)"
+  ];
+
+  NIX_CFLAGS_COMPILE = [
+    ''-UROOTLIBEXECDIR'' ''-DROOTLIBEXECDIR="/run/current-system/systemd/lib/systemd"''
+    ''-USYSTEMD_BINARY_PATH'' ''-DSYSTEMD_BINARY_PATH="/run/current-system/systemd/lib/systemd/systemd"''
+  ];
+
+  installFlags = [
+    "sysconfdir=$(out)/etc"
+  ];
+
+  preConfigure = ''
+    ./autogen.sh
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "http://www.freedesktop.org/wiki/Software/systemd";
```

Comment:
I wonder if https://github.com/systemd/systemd-bootchart might be better here, given that they've [split it explicitly *"because it is not actually related to systemd in any way"*](https://github.com/systemd/systemd-bootchart/commit/e6c5e467c4b593114a4cefa17ed10afd6def1d6e)